yaptide/utils/sim_utils.py

```diff
@@ -13,6 +14,9 @@
 from ..converter.converter.api import get_parser_from_str, run_parser  # skipcq: FLK-E402
 
 
+NSTAT_MATCH = r"NSTAT\s*\d*\s*\d*\s*!\s*NSTAT,\s*Step\s*of\s*saving"
```

Comment:
This regexp is not correct,
user may as well have in `beam.dat` simply:

```
NSTAT 1000 10
```

Comments are not obligatory

Reply:
Changed